src/applications/gi/tests/e2e/00-required.cypress.spec.js

```diff
@@ -0,0 +1,69 @@
+import { initMockProfile, expectLocation } from './cypress-helpers';
+import institutionProfile from '../data/institution-profile.json';
+import searchResults from '../data/search-results.json';
+import { createId } from '../../utils/helpers';
+import {
+  typeOfInstitution,
+  search,
+  selectSearchResult,
+  displayLearnMoreModal,
+  collapseExpandAccordion,
+} from './gi-helpers';
+
+const institutionAttributes = institutionProfile.data.attributes;
+
+describe('Institution', () => {
+  beforeEach(() => {
+    cy.route('/v0/gi/institutions/', searchResults);
+
+    initMockProfile(institutionProfile);
+    cy.visit('/gi-bill-comparison-tool').injectAxe();
+    cy.axeCheck();
+  });
+
+  it('Default institution profile flow with giBillChapter chapter 33', () => {
+    // Landing Page
+    typeOfInstitution('school');
+
+    cy.axeCheck();
+    cy
+      .get('.keyword-search input[type="text"]')
+      .type(institutionAttributes.name).type;
```

Comment:
can you pull this out into a method in `gi-helpers.js` or update the search method to take a parameter and if parameter is present do this?